.github/workflows/bump-golang.yml

```diff
@@ -21,8 +21,32 @@ jobs:
           command: --experimental apply --config .ci/bump-golang.yml
         env:
           GITHUB_TOKEN: ${{ secrets.UPDATECLI_GH_TOKEN }}
+          GITHUB_BRANCH: 'main'
 
-      - if: ${{ failure() }}
+  bump-7:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          ref: '7.17'
```

Comment:
This is the reason I cannot test my changes using a feature branch and I also need to backport the changes to `7.17`.